code/extract_transform.py

```diff
@@ -0,0 +1,33 @@
+import json
+import pandas as pd
+
+
+# def load_json(data):
+    # Load JSON
+#x=pd.DataFrame()
+def load():
+    with open("../json/top-rated-movies-02.json") as f:
```

Comment:
I wouldn't hardcode names, rather would pass the file name as a parameter.

Reply:
moved the same to dev.ini